mongo/options/findoptions.go

```diff
@@ -230,144 +275,194 @@ func MergeFindOptions(opts ...*FindOptions) *FindOptions {
+
+	// Specifies a collation to use for string comparisons during the operation. This option is only valid for MongoDB
+	// versions >= 3.4. For previous server versions, the driver will return an error if this option is used. The
+	// default value is nil, which means the default collation of the collection will be used.
+	Collation *Collation
+
+	// A string that will be included in server logs, profiling logs, and currentOp queries to help trace the operation.
+	// The default is the empty string, which means that no comment will be included in the logs.
+	Comment *string
+
+	// TODO
+	CursorType *CursorType
+
+	// The index to use for the aggregation. This should either be the index name as a string or the index specification
+	// as a document. The default value is nil, which means that no hint will be sent.
+	Hint interface{}
+
+	// A document specifying the exclusive upper bound for a specific index. The default value is nil, which means that
+	// there is no maximum value.
+	Max interface{}
+
+	// The maximum amount of time that the server should wait for new documents to satisfy a tailalbe cursor query.
+	// This option is only valid for tailable await cursors (see the CursorType option for more information) and
+	// MongoDB versions >= 3.2. For other cursor types or previous server versions, this option is ignored.
+	MaxAwaitTime *time.Duration
+
+	// The maximum amount of time that the query can run on the server. The default value is nil, meaning that there
+	// is no time limit for query execution.
+	MaxTime *time.Duration
+
+	// A document specifying the inclusive lower bound for a specific index. The default value is 0, which means that
+	// there is no minimum value.
+	Min interface{}
+
+	// If true, the cursor created by the operation will not timeout after a period of inactivity. The default value
+	// is false.
+	NoCursorTimeout *bool
+
+	// This option is for internal replication use only and should not be set.
+	OplogReplay *bool
+
+	// A document describing which fields will be included in the document returned by the operation. The default value
+	// is nil, which means all fields will be included.
+	Projection interface{}
+
+	// If true, the document returned by the operation will only contain fields corresponding to the index used. The
+	// default value is false.
+	ReturnKey *bool
+
+	// If true, a $recordId field with a record identifier will be included in the document returned by the operation.
+	// The default value is false.
+	ShowRecordID *bool
+
+	// The number of documents to skip before adding selecting the document to be returned. The default value is 0.
+	Skip *int64
+
+	// If true, the cursor will not return a document more than once because of an intervening write operation. This
+	// option has been deprecated in MongoDB version 4.0. The default value is false.
+	Snapshot *bool
+
+	// A document specifying the order in which documents should be should be sorted before a document to be returned
```

Comment:
Phrasing is awkward.  Perhaps

   A document specifying the sort order to apply to the query.  The first document in sorted order will be returned.

Reply:
Done.